doc/stdlib/linalg.rst

```diff
@@ -733,8 +833,7 @@ Linear algebra functions in Julia are largely implemented by calling functions f
 
    For more information, see [issue8859]_, [B96]_, [S84]_, [KY88]_.
 
-   .. [issue8859] Issue 8859, "Fix least squares",
-      https://github.com/JuliaLang/julia/pull/8859
+   .. [issue8859] Issue 8859, "Fix least squares", https://github.com/JuliaLang/julia/pull/8859
```

Comment:
@StefanKarpinski This is updated automatically by the script so I assume it's valid. Just to let you know in case it's not.